tests/kernel/install_ltp.pm

```diff
@@ -25,16 +25,16 @@ sub add_repos {
     zypper_call('--gpg-auto-import-keys ref', dumb_term => 1);
 }
 
+# SLE as registered product
 sub scc_we_enabled {
     script_run("zypper -n products -i", 0);
     return wait_serial(qr/Workstation Extension/);
 }
 
 sub add_we_repo_if_available {
-    # opensuse (doesn't have extensions) or SLE as registered product
-    if (check_var('DISTRI', 'opensuse') || scc_we_enabled) {
```

Comment:
Is there a case where we boot from an disk image which already have the workstation enabled? If not, this change is ok

Reply:
@pevik, this is still not answered

Reply:
no , we boot only images with pure SLES or SLES+ LP
WE or SLED covers only products with General Support and only on x86_64 so we don't test this path. 
+ ntfs isnt so interesting for us